packages/bot/src/stores/quick-strategy-store.js

```diff
@@ -98,41 +110,128 @@ export default class QuickStrategyStore {
         this.toggleStrategyModal();
     }
 
+    validateQuickStrategy = values => {
+        const errors = {};
+        const number_field = ['duration', 'stake', 'size', 'profit', 'loss'];
+    
+        Object.keys(values).forEach(key => {
+            const value = values[key];
+    
+            if (number_field.includes(key)){
+                if (isNaN(value)) {
+                    errors[key] = translate('Must be a number');
+                }
+    
+                if (value <= 0){
+                    errors[key] = translate('Must be a number higher than 0');
+                }
+    
+                if (/^0+(?=\d)/.test(value)) {
+                    errors[key] = translate('Invalid number format');
+                }
+            }
+    
+            if (value === '') {
+                errors[key] = translate('Field cannot be empty');
+            }
+        });
+
+        const { min, max } = this.duration_dropdown.filter(duration => duration.unit === values.duration_type)[0];
+        if (values.duration < min) {
+            errors.duration = `${translate('Minimum duration :')} ${min}`;
+        }
+
+        if (values.duration > max) {
+            errors.duration = `${translate('Maximum duration :')} ${max}`;
```

Comment:
- Make sure to use `localize` here
- There's a space after duration

```suggestion
            errors.duration = `${translate('Maximum duration:')} ${max}`;
```